os/board/rtl8730e/src/rtl8730e_st7789.c

```diff
@@ -92,14 +106,18 @@ static void gpio_pins_init(void)
 	CmdPin.GPIO_PuPd = GPIO_PuPd_NOPULL;
 	CmdPin.GPIO_Mode = GPIO_Mode_OUT;
 	GPIO_Init(&CmdPin);
+
+	pwmout_init(&g_rtl8730_config_dev_s.pwm_led, GPIO_PIN_BACKLIGHT);
+
+	GPIO_WriteBit(GPIO_PIN_BACKLIGHT, 1);
```

Comment:
That pin is inited as pwm. Is it ok to call GPIO API after that?
@edwakuwaku Is it ok in BSP?

Reply:
What does it mean `1`? Backlight on?

Reply:
Theoretically, `pmwout_init` and `pwmout_write` should be enough.
I think you can remove these few lines:
```
	Pinmux_Config(GPIO_PIN_BACKLIGHT, PINMUX_FUNCTION_GPIO);
	RSPin.GPIO_Pin = GPIO_PIN_BACKLIGHT;
	RSPin.GPIO_PuPd = GPIO_PuPd_NOPULL;
	RSPin.GPIO_Mode = GPIO_Mode_OUT;
	GPIO_Init(&RSPin);
```
and also remove 
`GPIO_WriteBit(GPIO_PIN_BACKLIGHT, 1);`

In `pwmout_init`, it will invoke `Pinmux_Config `to overwrite the function assigned to the pin.
Which means `Pinmux_Config(GPIO_PIN_BACKLIGHT, PINMUX_FUNCTION_GPIO);` will not be useful.

Could you try again with removing the few lines as stated above? It should work fine for PWM without invoking GPIO API.

The 1 in `GPIO_WriteBit `is to pull the pin state to high.

Reply:
yes , it works without gpio. I have updated PR